lib/developer_portal/lib/liquid/tags/active_docs.rb

```diff
@@ -21,14 +21,16 @@ def initialize(tag_name, params, tokens)
       end
 
       def render context
+        provider_account = context.registers[:controller].send(:site_account)
+        api_docs_services = provider_account.api_docs_services.published
+
         if @services.empty? && @version > "1.2"
-          @services =
-            context.registers[:controller].send(:site_account)
-            .api_docs_services.published
-            .select{|e| e.specification.swagger_2_0?}.map(&:system_name)
+          @services = api_docs_services.select{|e| e.specification.swagger_2_0?}.map(&:system_name)
         end
 
-        render_erb context, "shared/#{version}", {services: @services}
+        @api_docs_services = api_docs_services.where(system_name: @services)
+
+        render_erb context, "shared/#{version}", {services: @services, api_docs_services: @api_docs_services}
```

Comment:
why do you need that?

```liquid
{% for service in services %}
  {% for spec in service.api_specs %}
  // do whatever
  {% endfor %}
{% endfor %}

```

Reply:
There is also https://github.com/3scale/porta/blob/5b9187805193e4bb8b696c21df9edb6af1e7889a/lib/developer_portal/lib/liquid/drops/provider.rb#L202-L212

`provider` is available everywhere

Reply:
@hallelujah, I think `services` is an array of service system names.

Reply:
Oh services are the api_spec system_name!!! 🤦‍♂️ 
Anyway I do not think we should add it here. You can always use  `provider.api_specs` and filter

Reply:
@guicassolato added this to make `@api_docs_services` available from **_swagger_*.html.erb**, so that we have access to `host_with_port`. I don't understand what your comments have to do with this.